core/modules/simpletest/simpletest.module

```diff
@@ -372,13 +372,13 @@ function simpletest_test_get_all() {
           $error = 'File %uri contains no testing classes.';
         }
         else {
+          // If the .tests.info file is malformed, it puts a slew of
+          // hard-to-diagnose errors into the watchdog log when the testing page
+          // is loaded. So we'll check for various malformalities and put a more
+          // informative error.
           foreach ($classes as $class => $info) {
-            // If the .tests.info file is malformed, it puts a slew of
-            // hard-to-diagnose errors into the watchdog log when the testing page
-            // is loaded. So we'll check for various malformalities and put a more
-            // informative error.
             if (!is_array($info) || empty($info)) {
-              $error = format_string('File %uri contains a definition for %class that is not an array of testing class fields.', array('%class' => $class));
+              $error = t('File %uri contains a definition for %class that is not an array of testing class fields.', array('%class' => $class));
```

Comment:
I'm curious: why switching from format_string to t()?

Reply:
I realized these error messages show up in the UI if one is running Simpletest from the UI, so seemed like they should have been translated all along.